Marlin/ultralcd.cpp

```diff
@@ -670,7 +670,7 @@ static void lcd_move_menu_axis();
 
 static void _lcd_move(const char *name, int axis, int min, int max) {
   if (encoderPosition != 0) {
-    refresh_cmd_timeout();
+    void refresh_cmd_timeout();
```

Comment:
It already was from the update you pushed...still didn't work. I made this change and it fixed it.